target_chains/ethereum/contracts/contracts/entropy/EntropyState.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: Apache 2
```

Comment:
very minor change here: we were exposing the state object in the public SDK. I moved that definition into the contract directory instead.